types/database.go

```diff
@@ -16,6 +18,14 @@ func (c *DatabaseContainer) Get(cloudType CloudType) []*Database {
 	return items
 }
 
+func (d *DatabaseContainer) String() string {
+	var result string
```

Comment:
```suggestion
	var buffer bytes.Buffer
	for _, db := range d.databases {
		buffer.WriteString(db.Name+ " ")
	}
	return buffer.String()
```

:D 

Does the extra space at the end cause any trouble?

Reply:
Mypleasere @keyki , i'm fine with it ;) I just wanted to say hello to you on the programmer way :D

Reply:
Applied your suggestion, though :)